CHANGES.rst

```diff
@@ -0,0 +1,66 @@
+Changelog
+=========
+
+Version 0.1.0 [2020-01-15]
+--------------------------
+
+- [deps] Verified support for python 3.8
+- [deps] Added support for Django 3.0 and Django Rest Framework 3.11
+
+Version 1.1.1 [2019-07-23]
+--------------------------
+
+- [deps] Drop python<3.3 support
+- [deps] Added support for python 3.7
+- [deps] Django 2 support added
+
+Version 1.1.0 [2017-05-11]
+--------------------------
+
+- [test] Added tests for swapper.split
+- #13 [fix] Handle contrib apps and apps with dot in app_label.
+
+Version 1.0.0 [2016-08-26]
+--------------------------
+
+- [docs] Improved usuability docs
+- `86e238 <https://github.com/openwisp/django-swappable-models/commit/86e238>`_:
+  [deps] Compatibility with django 1.10 added
+
+Version 0.3.0 [2015-11-17]
+--------------------------
+
+- #9 [deps] Added support for django 1.9
+
+Version 0.2.2 [2015-06-16]
+--------------------------
+
+- [deps] Added support for django~=1.6.0
+- [deps] Added support for Python 3.3
+- [docs] Fix model reference in README
+- #2 [docs] Notes for load_model initialization
```

Comment:
Is this `#2` here intentional?

Reply:
Yes, I use #2 here because it was explained in that issue.

Reply:
can you change it to something like the following?
```
[docs] Notes for load_model initialization (`for more info see #2 <issue_url>`_)
```